--- gaim-2.0.0beta1/src/account.c.orig	2005-12-08 22:50:49.000000000 +0100
+++ gaim-2.0.0beta1/src/account.c	2005-12-19 20:48:30.000000000 +0100
@@ -1384,15 +1384,12 @@
 	const gchar *id;
 	gpointer data;
 
-	if (args != NULL)
-	{
 		while ((id = va_arg(args, const char *)) != NULL)
 		{
 			attrs = g_list_append(attrs, (char *)id);
 			data = va_arg(args, void *);
 			attrs = g_list_append(attrs, data);
 		}
-	}
 	gaim_account_set_status_list(account, status_id, active, attrs);
 	g_list_free(attrs);
 }
--- gaim-2.0.0beta1/plugins/perl/common/Account.xs.orig	2005-12-06 00:54:34.000000000 +0100
+++ gaim-2.0.0beta1/plugins/perl/common/Account.xs	2005-12-19 20:48:00.000000000 +0100
@@ -89,7 +89,7 @@
 	const char *status_id
 	gboolean active
 CODE:
-	gaim_account_set_status(account, status_id, active);
+	gaim_account_set_status(account, status_id, active, NULL);
 
 
 void 
--- gaim-2.0.0beta1/src/status.c.orig	2005-12-09 14:16:09.000000000 +0100
+++ gaim-2.0.0beta1/src/status.c	2005-12-19 21:13:22.000000000 +0100
@@ -699,7 +699,7 @@
 void
 gaim_status_set_active(GaimStatus *status, gboolean active)
 {
-	gaim_status_set_active_with_attrs(status, active, NULL);
+	gaim_status_set_active_with_attrs_list(status, active, NULL);
 }
 
 /*
@@ -715,15 +715,12 @@
 	const gchar *id;
 	gpointer data;
 
-	if (args != NULL)
-	{
 		while ((id = va_arg(args, const char *)) != NULL)
 		{
 			attrs = g_list_append(attrs, (char *)id);
 			data = va_arg(args, void *);
 			attrs = g_list_append(attrs, data);
 		}
-	}
 	gaim_status_set_active_with_attrs_list(status, active, attrs);
 	g_list_free(attrs);
 }
