diff -urN gaim-0.44.orig/src/protocols/icq/Makefile.am gaim-0.44/src/protocols/icq/Makefile.am
--- gaim-0.44.orig/src/protocols/icq/Makefile.am	Tue Jul 31 03:58:50 2001
+++ gaim-0.44/src/protocols/icq/Makefile.am	Sat Sep 29 15:55:13 2001
@@ -9,6 +9,8 @@
 
 CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
+libicq_la_LDFLAGS =	-avoid-version
+
 if STATIC_ICQ
 
 st = -DSTATIC
diff -urN gaim-0.44.orig/src/protocols/irc/Makefile.am gaim-0.44/src/protocols/irc/Makefile.am
--- gaim-0.44.orig/src/protocols/irc/Makefile.am	Sun Sep 16 09:05:14 2001
+++ gaim-0.44/src/protocols/irc/Makefile.am	Sat Sep 29 15:55:21 2001
@@ -4,6 +4,8 @@
 
 CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
+libirc_la_LDFLAGS = -avoid-version
+
 if STATIC_IRC
 
 st = -DSTATIC
diff -urN gaim-0.44.orig/src/protocols/jabber/Makefile.am gaim-0.44/src/protocols/jabber/Makefile.am
--- gaim-0.44.orig/src/protocols/jabber/Makefile.am	Tue Jul 31 03:00:38 2001
+++ gaim-0.44/src/protocols/jabber/Makefile.am	Sat Sep 29 15:55:31 2001
@@ -4,6 +4,8 @@
 
 CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
+libjabber_la_LDFLAGS = 	-avoid-version
+
 if STATIC_JABBER
 
 st = -DSTATIC
diff -urN gaim-0.44.orig/src/protocols/msn/Makefile.am gaim-0.44/src/protocols/msn/Makefile.am
--- gaim-0.44.orig/src/protocols/msn/Makefile.am	Sun Sep 16 09:05:14 2001
+++ gaim-0.44/src/protocols/msn/Makefile.am	Sat Sep 29 15:55:38 2001
@@ -4,6 +4,8 @@
 
 CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
+libmsn_la_LDFLAGS = -avoid-version
+
 if STATIC_MSN
 
 st = -DSTATIC
diff -urN gaim-0.44.orig/src/protocols/napster/Makefile.am gaim-0.44/src/protocols/napster/Makefile.am
--- gaim-0.44.orig/src/protocols/napster/Makefile.am	Tue Jul 31 03:00:38 2001
+++ gaim-0.44/src/protocols/napster/Makefile.am	Sat Sep 29 15:55:45 2001
@@ -2,6 +2,8 @@
 
 CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
+libnapster_la_LDFLAGS = -avoid-version
+
 if STATIC_NAPSTER
 
 st = -DSTATIC
diff -urN gaim-0.44.orig/src/protocols/oscar/Makefile.am gaim-0.44/src/protocols/oscar/Makefile.am
--- gaim-0.44.orig/src/protocols/oscar/Makefile.am	Tue Jul 31 03:00:38 2001
+++ gaim-0.44/src/protocols/oscar/Makefile.am	Sat Sep 29 15:55:57 2001
@@ -5,6 +5,8 @@
 
 CFLAGS += -DAIM_BUILDDATE=\"`date +%Y%m%d`\" -DAIM_BUILDTIME=\"`date +%H%M%S`\" -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
+liboscar_la_LDFLAGS =	-avoid-version
+
 if STATIC_OSCAR
 
 st = -DSTATIC
diff -urN gaim-0.44.orig/src/protocols/toc/Makefile.am gaim-0.44/src/protocols/toc/Makefile.am
--- gaim-0.44.orig/src/protocols/toc/Makefile.am	Sun Sep 16 09:05:14 2001
+++ gaim-0.44/src/protocols/toc/Makefile.am	Sat Sep 29 15:56:04 2001
@@ -4,6 +4,8 @@
 
 CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
+libtoc_la_LDFLAGS = -avoid-version
+
 if STATIC_TOC
 
 st = -DSTATIC
diff -urN gaim-0.44.orig/src/protocols/yahoo/Makefile.am gaim-0.44/src/protocols/yahoo/Makefile.am
--- gaim-0.44.orig/src/protocols/yahoo/Makefile.am	Tue Jul 31 03:00:38 2001
+++ gaim-0.44/src/protocols/yahoo/Makefile.am	Sat Sep 29 15:56:13 2001
@@ -4,6 +4,8 @@
 
 CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
+libyahoo_la_LDFLAGS = -avoid-version
+
 if STATIC_YAHOO
 
 st = -DSTATIC
diff -urN gaim-0.44.orig/src/protocols/zephyr/Makefile.am gaim-0.44/src/protocols/zephyr/Makefile.am
--- gaim-0.44.orig/src/protocols/zephyr/Makefile.am	Sat Sep 29 15:49:13 2001
+++ gaim-0.44/src/protocols/zephyr/Makefile.am	Sat Sep 29 15:56:29 2001
@@ -4,6 +4,8 @@
 
 LIBS = @LIBS@ $(KRB4_LDFLAGS) $(KRB4_LIBS)
 
+libzephyr_la_LDFLAGS = 	-avoid-version
+
 if STATIC_ZEPHYR
 
 st = -DSTATIC
