--- gaim-0.80/configure.ac~	Sat Jul 24 16:55:36 2004
+++ gaim-0.80/configure.ac	Sat Jul 24 16:55:36 2004
@@ -33,7 +33,7 @@
 	;;
 esac
 
-ALL_LINGUAS="am bg ca cs da de en_CA en_GB es fi fr he hi hu it ja ko lt mk nl no pl pt_BR pt ro ru sk sl sr sr@Latn sv vi zh_CN zh_TW"
+ALL_LINGUAS="am bg ca cs da de en_CA en_GB es fi fr he hi hu it ja ko lt mk nb nl pl pt_BR pt ro ru sk sl sr sr@Latn sv vi zh_CN zh_TW"
 AM_GNU_GETTEXT_VERSION(0.10.40)
 AM_GNU_GETTEXT
 
--- gaim-0.80/configure~	Sat Jul 24 16:55:48 2004
+++ gaim-0.80/configure	Sat Jul 24 16:55:49 2004
@@ -19344,7 +19344,7 @@
 	;;
 esac
 
-ALL_LINGUAS="am bg ca cs da de en_CA en_GB es fi fr he hi hu it ja ko lt mk nl no pl pt_BR pt ro ru sk sl sr sr@Latn sv vi zh_CN zh_TW"
+ALL_LINGUAS="am bg ca cs da de en_CA en_GB es fi fr he hi hu it ja ko lt mk nb nl pl pt_BR pt ro ru sk sl sr sr@Latn sv vi zh_CN zh_TW"
 
 
         MKINSTALLDIRS=
