--- gaim-2.0.0beta2/configure.ac~	2006-02-07 18:49:47.000000000 +0200
+++ gaim-2.0.0beta2/configure.ac	2006-02-07 19:00:20.000000000 +0200
@@ -540,10 +540,6 @@
 if test "x$enable_dbus" = "xyes" ; then
     AC_MSG_CHECKING([location of the D-BUS session directory])
     if ! test -z "$with_dbus_session_dir"; then
-		if ! test -d "$with_dbus_session_dir"; then
-			AC_MSG_WARN([$with_dbus_session_dir does not exist, if this is the correct location please make sure that it exists.])
-			enable_dbus=no
-		fi
 		DBUS_SESSION_DIR="$with_dbus_session_dir"
     else
 	dnl # add more to this as needed
