--- gaim-0.56/src/protocols/gg/Makefile.am.wiget	Wed Apr 17 10:51:06 2002
+++ gaim-0.56/src/protocols/gg/Makefile.am	Wed Apr 17 14:57:10 2002
@@ -2,27 +2,24 @@
 
 pkgdir = $(libdir)/gaim
 
-CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
-libgg_la_LDFLAGS = -module -avoid-version
+AM_CFLAGS = -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
+
 
 if STATIC_GG
 
 st = -DSTATIC
 pkg_LTLIBRARIES =
-noinst_LIBRARIES = libgg.a
-
-libgg_a_SOURCES = libgg.c \
-		  libgg.h \
-		  common.c \
-		  iconv_string.c \
-		  iconv_string.h \
-		  gg.c
+noinst_LTLIBRARIES = libgg-static.la
+libgg_la_LDFLAGS =
 
 else
 
 st =
 pkg_LTLIBRARIES = libgg.la
-noinst_LIBRARIES =
+noinst_LTLIBRARIES =
+libgg_la_LDFLAGS = -module -avoid-version
+
+endif
 
 libgg_la_SOURCES = libgg.c \
 		   libgg.h \
@@ -30,5 +27,6 @@
 		   iconv_string.c \
 		   iconv_string.h \
 		   gg.c
+		   
+libgg_static_la_SOURCES = $(libgg_la_SOURCES)
 
-endif
--- gaim-0.56/src/protocols/irc/Makefile.am.wiget	Wed Apr 17 11:17:13 2002
+++ gaim-0.56/src/protocols/irc/Makefile.am	Wed Apr 17 13:15:31 2002
@@ -2,23 +2,24 @@
 
 pkgdir = $(libdir)/gaim
 
-CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
-libirc_la_LDFLAGS = -module -avoid-version
+AM_CFLAGS = -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
 if STATIC_IRC
 
 st = -DSTATIC
 pkg_LTLIBRARIES =
-noinst_LIBRARIES = libirc.a
-
-libirc_a_SOURCES = irc.c
+noinst_LTLIBRARIES = libirc-static.la
+libirc_la_LDFLAGS = 
 
 else
 
 st =
 pkg_LTLIBRARIES = libirc.la
 noinst_LIBRARIES =
+libirc_la_LDFLAGS = -module -avoid-version
+
+endif
 
 libirc_la_SOURCES = irc.c
 
-endif
+libirc_static_la_SOURCES = $(libirc_la_SOURCES)
--- gaim-0.56/src/protocols/jabber/Makefile.am.wiget	Wed Apr 17 12:23:09 2002
+++ gaim-0.56/src/protocols/jabber/Makefile.am	Wed Apr 17 13:15:21 2002
@@ -2,62 +2,23 @@
 
 pkgdir = $(libdir)/gaim
 
-CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
-libjabber_la_LDFLAGS = 	-module -avoid-version
+AM_CFLAGS = -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
 if STATIC_JABBER
 
 st = -DSTATIC
 pkg_LTLIBRARIES =
-noinst_LIBRARIES = libjabber.a
-
-libjabber_a_SOURCES = \
-			asciitab.h \
-			expat.c \
-			genhash.c \
-			hashtable.c \
-			hashtable.h \
-			karma.c \
-			iasciitab.h \
-			latin1tab.h \
-			lib.h \
-			libxode.h \
-			nametab.h \
-			pool.c \
-			sha.c \
-			snprintf.c \
-			socket.c \
-			str.c \
-			utf8tab.h \
-			xhash.c \
-			xmldef.h \
-			xmlnode.c \
-			xmlparse.c \
-			xmlparse.h \
-			xmlrole.c \
-			xmlrole.h \
-			xmltok.c \
-			xmltok.h \
-			xmltok_impl.h \
-			xstream.c \
-			\
-			jabber.h \
-			jconn.c \
-			jid.c \
-			jpacket.c \
-			jutil.c \
-			log.c \
-			log.h \
-			pproxy.c \
-			rate.c \
-			\
-			jabber.c
+noinst_LTLIBRARIES = libjabber-static.la
+libjabber_la_LDFLAGS = 
 
 else
 
 st =
 pkg_LTLIBRARIES = libjabber.la
-noinst_LIBRARIES =
+noinst_LTLIBRARIES =
+libjabber_la_LDFLAGS = 	-module -avoid-version
+
+endif
 
 libjabber_la_SOURCES = \
 			asciitab.h \
@@ -101,4 +62,4 @@
 			\
 			jabber.c
 
-endif
+libjabber_static_la_SOURCES = $(libjabber_la_SOURCES)
--- gaim-0.56/src/protocols/msn/Makefile.am.wiget	Wed Apr 17 12:24:06 2002
+++ gaim-0.56/src/protocols/msn/Makefile.am	Wed Apr 17 13:15:06 2002
@@ -2,23 +2,24 @@
 
 pkgdir = $(libdir)/gaim
 
-CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
-libmsn_la_LDFLAGS = -module -avoid-version
+AM_CFLAGS = -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
 if STATIC_MSN
 
 st = -DSTATIC
 pkg_LTLIBRARIES =
-noinst_LIBRARIES = libmsn.a
-
-libmsn_a_SOURCES = msn.c
+noinst_LTLIBRARIES = libmsn-static.la
+libmsn_la_LDFLAGS = 
 
 else
 
 st =
 pkg_LTLIBRARIES = libmsn.la
-noinst_LIBRARIES =
+noinst_LTLIBRARIES =
+libmsn_la_LDFLAGS = -module -avoid-version
+
+endif
 
 libmsn_la_SOURCES = msn.c
 
-endif
+libmsn_static_la_SOURCES = $(libmsn_la_SOURCES)
--- gaim-0.56/src/protocols/napster/Makefile.am.wiget	Wed Apr 17 12:25:01 2002
+++ gaim-0.56/src/protocols/napster/Makefile.am	Wed Apr 17 13:14:34 2002
@@ -1,22 +1,23 @@
 pkgdir = $(libdir)/gaim
 
-CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
-libnapster_la_LDFLAGS = -module -avoid-version
+AM_CFLAGS = -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
 if STATIC_NAPSTER
 
 st = -DSTATIC
 pkg_LTLIBRARIES =
-noinst_LIBRARIES = libnapster.a
-
-libnapster_a_SOURCES = napster.c
+noinst_LTLIBRARIES = libnapster-static.la
+libnapster_la_LDFLAGS =
 
 else
 
 st =
 pkg_LTLIBRARIES = libnapster.la
-noinst_LIBRARIES =
+noinst_LTLIBRARIES =
+libnapster_la_LDFLAGS = -module -avoid-version
+
+endif
 
 libnapster_la_SOURCES = napster.c
 
-endif
+libnapster_static_la_SOURCES = $(libnapster_la_SOURCES)
--- gaim-0.56/src/protocols/oscar/Makefile.am.wiget	Wed Apr 17 12:18:01 2002
+++ gaim-0.56/src/protocols/oscar/Makefile.am	Wed Apr 17 13:16:00 2002
@@ -3,51 +3,23 @@
 
 pkgdir = $(libdir)/gaim
 
-CFLAGS += -DAIM_BUILDDATE=\"`date +%Y%m%d`\" -DAIM_BUILDTIME=\"`date +%H%M%S`\" -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
-liboscar_la_LDFLAGS =	-module -avoid-version
+AM_CFLAGS = -DAIM_BUILDDATE=\"`date +%Y%m%d`\" -DAIM_BUILDTIME=\"`date +%H%M%S`\" -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
 if STATIC_OSCAR
 
 st = -DSTATIC
 pkg_LTLIBRARIES =
-noinst_LIBRARIES = liboscar.a
-
-liboscar_a_SOURCES = 	admin.c		\
-			adverts.c	\
-			auth.c 		\
-			bos.c 		\
-			buddylist.c	\
-			chat.c		\
-			chatnav.c	\
-			conn.c		\
-			ft.c		\
-			icq.c		\
-			im.c		\
-			info.c		\
-			invite.c	\
-			meta.c		\
-			misc.c		\
-			msgcookie.c	\
-			popups.c	\
-			rxhandlers.c	\
-			rxqueue.c	\
-			search.c	\
-			service.c	\
-			snac.c		\
-			ssi.c		\
-			stats.c		\
-			tlv.c		\
-			translate.c	\
-			txqueue.c	\
-			util.c		\
-			oscar.c
-
+noinst_LTLIBRARIES = liboscar-static.la
+liboscar_la_LDFLAGS =
 
 else
 
 st =
 pkg_LTLIBRARIES = liboscar.la
-noinst_LIBRARIES =
+noinst_LTLIBRARIES =
+liboscar_la_LDFLAGS =	-module -avoid-version
+
+endif
 
 liboscar_la_SOURCES = 	admin.c		\
 			adverts.c	\
@@ -79,4 +51,4 @@
 			util.c		\
 			oscar.c
 
-endif
+liboscar_static_la_SOURCES = $(liboscar_la_SOURCES)
--- gaim-0.56/src/protocols/toc/Makefile.am.wiget	Wed Apr 17 11:27:30 2002
+++ gaim-0.56/src/protocols/toc/Makefile.am	Wed Apr 17 13:16:30 2002
@@ -2,23 +2,24 @@
 
 pkgdir = $(libdir)/gaim
 
-CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
-libtoc_la_LDFLAGS = -module -avoid-version
+AM_CFLAGS = -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
 if STATIC_TOC
 
 st = -DSTATIC
 pkg_LTLIBRARIES =
-noinst_LIBRARIES = libtoc.a
-
-libtoc_a_SOURCES = toc.c
+noinst_LTLIBRARIES = libtoc-static.la
+libtoc_la_LDFLAGS =
 
 else
 
 st =
 pkg_LTLIBRARIES = libtoc.la
-noinst_LIBRARIES =
+noinst_LTLIBRARIES =
+libtoc_la_LDFLAGS = -module -avoid-version
+
+endif
 
 libtoc_la_SOURCES = toc.c
 
-endif
+libtoc_static_la_SOURCES = $(libtoc_la_SOURCES)
--- gaim-0.56/src/protocols/yahoo/Makefile.am.wiget	Wed Apr 17 11:26:03 2002
+++ gaim-0.56/src/protocols/yahoo/Makefile.am	Wed Apr 17 13:17:12 2002
@@ -2,23 +2,24 @@
 
 pkgdir = $(libdir)/gaim
 
-CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
-libyahoo_la_LDFLAGS = -module -avoid-version
+AM_CFLAGS = -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
 if STATIC_YAHOO
 
 st = -DSTATIC
 pkg_LTLIBRARIES =
-noinst_LIBRARIES = libyahoo.a
-
-libyahoo_a_SOURCES = crypt.c yahoo.c
+noinst_LTLIBRARIES = libyahoo-static.la
+libyahoo_la_LDFLAGS =
 
 else
 
 st =
 pkg_LTLIBRARIES = libyahoo.la
-noinst_LIBRARIES =
+noinst_LTLIBRARIES =
+libyahoo_la_LDFLAGS = -module -avoid-version
+
+endif
 
 libyahoo_la_SOURCES = crypt.c yahoo.c
 
-endif
+libyahoo_static_la_SOURCES = $(libyahoo_la_SOURCES)
--- gaim-0.56/src/protocols/zephyr/Makefile.am.wiget	Wed Apr 17 11:18:58 2002
+++ gaim-0.56/src/protocols/zephyr/Makefile.am	Wed Apr 17 13:17:45 2002
@@ -1,86 +1,24 @@
 pkgdir = $(libdir)/gaim
 
-CFLAGS += -I\$(top_srcdir)/src -I\$(top_srcdir)/src/protocols -DCONFDIR=\"$(confdir)\" $(KRB4_CFLAGS) $(st) $(DEBUG_CFLAGS)
+AM_CFLAGS = -I\$(top_srcdir)/src -I\$(top_srcdir)/src/protocols -DCONFDIR=\"$(confdir)\" $(KRB4_CFLAGS) $(st) $(DEBUG_CFLAGS)
 
-LIBS = @LIBS@ $(KRB4_LDFLAGS) $(KRB4_LIBS)
-libzephyr_la_LDFLAGS = 	-module -avoid-version
+libzephyr_la_LIBADD = $(KRB4_LDFLAGS) $(KRB4_LIBS)
 
 if STATIC_ZEPHYR
 
 st = -DSTATIC
 pkg_LTLIBRARIES =
-noinst_LIBRARIES = libzephyr.a
-
-libzephyr_a_SOURCES = \
-			ZAsyncLocate.c \
-			ZCkAuth.c \
-			ZCkIfNot.c \
-			ZClosePort.c \
-			ZCmpUID.c \
-			ZCmpUIDP.c \
-			ZFlsLocs.c \
-			ZFlsSubs.c \
-			ZFmtAuth.c \
-			ZFmtList.c \
-			ZFmtNotice.c \
-			ZFmtRaw.c \
-			ZFmtRawLst.c \
-			ZFmtSmRLst.c \
-			ZFmtSmRaw.c \
-			ZFreeNot.c \
-			ZGetLocs.c \
-			ZGetSender.c \
-			ZGetSubs.c \
-			ZGetWGPort.c \
-			ZIfNotice.c \
-			ZInit.c \
-			ZLocations.c \
-			ZMakeAscii.c \
-			ZMkAuth.c \
-			ZNewLocU.c \
-			ZOpenPort.c \
-			ZParseNot.c \
-			ZPeekIfNot.c \
-			ZPeekNot.c \
-			ZPeekPkt.c \
-			ZPending.c \
-			ZReadAscii.c \
-			ZRecvNot.c \
-			ZRecvPkt.c \
-			ZRetSubs.c \
-			ZSendList.c \
-			ZSendNot.c \
-			ZSendPkt.c \
-			ZSendRLst.c \
-			ZSendRaw.c \
-			ZSetDest.c \
-			ZSetFD.c \
-			ZSetSrv.c \
-			ZSubs.c \
-			ZVariables.c \
-			ZWait4Not.c \
-			ZhmStat.c \
-			Zinternal.c \
-			com_err.h \
-			error_message.c \
-			error_table.h \
-			et_name.c \
-			init_et.c \
-			internal.h \
-			mit-copyright.h \
-			mit-sipb-copyright.h \
-			sysdep.h \
-			zephyr.h \
-			zephyr_err.c \
-			zephyr_err.h \
-			\
-			zephyr.c
+noinst_LTLIBRARIES = libzephyr-static.la
+libzephyr_la_LDFLAGS = 
 
 else
 
 st =
 pkg_LTLIBRARIES = libzephyr.la
 noinst_LIBRARIES =
+libzephyr_la_LDFLAGS = 	-module -avoid-version
+
+endif
 
 libzephyr_la_SOURCES = \
 			ZAsyncLocate.c \
@@ -147,4 +85,4 @@
 			\
 			zephyr.c
 
-endif
+libzephyr_static_la_SOURCES = $(libzephyr_la_SOURCES)
--- gaim-0.56/src/protocols/icq/Makefile.am.wiget	Wed Apr 17 11:13:04 2002
+++ gaim-0.56/src/protocols/icq/Makefile.am	Wed Apr 17 13:12:08 2002
@@ -7,64 +7,23 @@
 
 pkgdir = $(libdir)/gaim
 
-CFLAGS += -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
-libicq_la_LDFLAGS =	-module -avoid-version
+AM_CFLAGS = -I\$(top_srcdir)/src $(st) $(DEBUG_CFLAGS)
 
 if STATIC_ICQ
 
 st = -DSTATIC
 pkg_LTLIBRARIES =
-noinst_LIBRARIES = libicq.a
-
-libicq_a_SOURCES = 	chatsession.c \
-			chatsession.h \
-			contacts.c \
-			contacts.h \
-			cyrillic.c \
-			eventhandle.c \
-			eventhandle.h \
-			filesession.c \
-			filesession.h \
-			icq.h \
-			icqbyteorder.c \
-			icqbyteorder.h \
-			icqevent.c \
-			icqevent.h \
-			icqlib.c \
-			icqlib.h \
-			icqpacket.c \
-			icqpacket.h \
-			icqtypes.h \
-			list.c \
-			list.h \
-			proxy.c \
-			queue.c \
-			queue.h \
-			socketmanager.c \
-			socketmanager.h \
-			stdpackets.c \
-			stdpackets.h \
-			tcp.c \
-			tcp.h \
-			tcpchathandle.c \
-			tcpfilehandle.c \
-			tcphandle.c \
-			tcplink.c \
-			tcplink.h \
-			timeout.c \
-			timeout.h \
-			udp.c \
-			udp.h \
-			udphandle.c \
-			util.c \
-			util.h \
-			gaim_icq.c
+noinst_LTLIBRARIES = libicq-static.la
+libicq_la_LDFLAGS =
 
 else
 
 st =
 pkg_LTLIBRARIES = libicq.la
-noinst_LIBRARIES =
+noinst_LTLIBRARIES =
+libicq_la_LDFLAGS =	-module -avoid-version
+
+endif
 
 libicq_la_SOURCES =	chatsession.c \
 			chatsession.h \
@@ -110,4 +69,4 @@
 			util.h \
 			gaim_icq.c
 
-endif
+libicq_static_la_SOURCES =	$(libicq_la_SOURCES)
--- gaim-0.56/configure.ac.wiget	Wed Apr 17 14:16:27 2002
+++ gaim-0.56/configure.ac	Wed Apr 17 14:28:02 2002
@@ -58,7 +58,7 @@
 extern_init=
 load_proto=
 for i in $STATIC_PRPLS ; do
-	STATIC_LINK_LIBS="$STATIC_LINK_LIBS protocols/$i/lib$i.a"
+	STATIC_LINK_LIBS="$STATIC_LINK_LIBS protocols/$i/lib$i-static.la"
 	extern_init="$extern_init extern void ${i}_init(struct prpl *);"
 	load_proto="$load_proto load_protocol(${i}_init, sizeof(struct prpl));"
 	case $i in
@@ -392,6 +392,10 @@
 AC_CHECK_HEADERS(sys/select.h sys/uio.h sys/utsname.h sys/wait.h)
 AC_CHECK_HEADERS(termios.h)
 
+dnl workaround
+AM_CONDITIONAL(HAVE_ORBIT, test x$gnome_cv_orbit_found = xyes)
+AM_CONDITIONAL(HAVE_GNORBA, test x$gnome_cv_gnorba_found = xyes)
+
 AC_OUTPUT([Makefile
            doc/Makefile
 	   intl/Makefile
