diff -urN gaim-0.45.org/src/gtkimhtml.c gaim-0.45/src/gtkimhtml.c
--- gaim-0.45.org/src/gtkimhtml.c	Sun Oct 14 19:46:14 2001
+++ gaim-0.45/src/gtkimhtml.c	Tue Oct 16 21:34:56 2001
@@ -19,6 +19,9 @@
  *
  */
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
 #include "gtkimhtml.h"
 #include <X11/Xlib.h>
 #include <gdk/gdkx.h>
@@ -27,6 +30,10 @@
 #include <ctype.h>
 #include <stdio.h>
 #include <math.h>
+#ifdef HAVE_LANGINFO_CODESET
+#include <langinfo.h>
+#include <locale.h>
+#endif
 
 #include "pixmaps/angel.xpm"
 #include "pixmaps/bigsmile.xpm"
@@ -79,6 +86,21 @@
 	gchar **image;
 };
 
+static gchar* getcharset()
+{
+  static gchar charset[64];
+#ifdef HAVE_LANGINFO_CODESET
+  gchar *ch = nl_langinfo(CODESET);
+  if (strncasecmp(ch, "iso-", 4) == 0)
+    g_snprintf(charset, sizeof(charset), "iso%s", ch + 4);
+  else
+    g_snprintf(charset, sizeof(charset), ch);
+#else
+  g_snprintf(charset, sizeof(charset), "iso8859-*");
+#endif
+  return charset;
+}
+
 static GtkSmileyTree*
 gtk_smiley_tree_new ()
 {
@@ -1781,53 +1803,60 @@
 		return gdk_fontset_load ("-*-*-*-*-*-*-*-*-*-*-*-*-*-*,*");
 	}
 
-	g_snprintf (buf, sizeof (buf), "-*-%s-%s-%c-*-*-*-%d-*-*-*-*-iso8859-*",
+	g_snprintf (buf, sizeof (buf), "-*-%s-%s-%c-*-*-*-%d-*-*-*-*-%s",
 		    choice,
 		    bold ? "bold" : "medium",
 		    italics ? 'i' : 'r',
-		    size);
+		    size,
+		    getcharset());
 	font = gdk_font_load (buf);
 
 	if (!font && italics) {
-		g_snprintf (buf, sizeof (buf), "-*-%s-%s-o-*-*-*-%d-*-*-*-*-iso8859-*",
+		g_snprintf (buf, sizeof (buf), "-*-%s-%s-o-*-*-*-%d-*-*-*-*-%s",
 			    choice,
 			    bold ? "bold" : "medium",
-			    size);
+			    size,
+			    getcharset());
 		font = gdk_font_load (buf);
 	}
 
 	if (!font) {
-		g_snprintf (buf, sizeof (buf), "-*-%s-%s-%c-*-*-*-*-*-*-*-*-iso8859-*",
+		g_snprintf (buf, sizeof (buf), "-*-%s-%s-%c-*-*-*-*-*-*-*-*-%s",
 			    choice,
 			    bold ? "bold" : "medium",
-			    italics ? 'i' : 'r');
+			    italics ? 'i' : 'r',
+			    getcharset());
 		font = gdk_font_load (buf);
 	}
 
 	if (!font && italics) {
-		g_snprintf (buf, sizeof (buf), "-*-%s-%s-o-*-*-*-*-*-*-*-*-iso8859-*",
+		g_snprintf (buf, sizeof (buf), "-*-%s-%s-o-*-*-*-*-*-*-*-*-%s",
 			    choice,
-			    bold ? "bold" : "medium");
+			    bold ? "bold" : "medium",
+			    getcharset());
 		font = gdk_font_load (buf);
 	}
 
 	if (!font) {
-		g_snprintf (buf, sizeof (buf), "-*-%s-*-%c-*-*-*-*-*-*-*-*-iso8859-*",
+		g_snprintf (buf, sizeof (buf), "-*-%s-*-%c-*-*-*-*-*-*-*-*-%s",
 			    choice,
-			    italics ? 'i' : 'r');
+			    italics ? 'i' : 'r',
+			    getcharset());
 		font = gdk_font_load (buf);
 	}
 
 	if (!font) {
-		g_snprintf (buf, sizeof (buf), "-*-%s-*-%c-*-*-*-*-*-*-*-*-iso8859-*",
+		g_snprintf (buf, sizeof (buf), "-*-%s-*-%c-*-*-*-*-*-*-*-*-%s",
 			    choice,
-			    italics ? 'o' : '*');
+			    italics ? 'o' : '*',
+			    getcharset());
 		font = gdk_font_load (buf);
 	}
 
 	if (!font && italics) {
-		g_snprintf (buf, sizeof (buf), "-*-%s-*-*-*-*-*-*-*-*-*-*-iso8859-*",
-			    choice);
+		g_snprintf (buf, sizeof (buf), "-*-%s-*-*-*-*-*-*-*-*-*-*-%s",
+			    choice,
+			    getcharset());
 		font = gdk_font_load (buf);
 	}
 
@@ -2032,6 +2061,11 @@
 	g_return_if_fail (imhtml != NULL);
 	g_return_if_fail (GTK_IS_IMHTML (imhtml));
 
+#ifdef HAVE_LANGINFO_CODESET
+	if (!font)
+	  font = gtk_imhtml_font_load (imhtml, "Helvetica", FALSE, FALSE, 0);
+#endif
+
 	if (font) {
 		if (imhtml->default_font)
 			gdk_font_unref (imhtml->default_font);
