diff -aurN gaim-2.0.0beta3.orig/src/Makefile.am gaim-2.0.0beta3/src/Makefile.am
--- gaim-2.0.0beta3.orig/src/Makefile.am	2006-03-25 16:45:49.000000000 +0100
+++ gaim-2.0.0beta3/src/Makefile.am	2006-08-10 22:59:08.308377751 +0200
@@ -198,6 +198,7 @@
 lib_LTLIBRARIES = libgaim-client.la
 
 libgaim_client_la_SOURCES = gaim-client.c gaim-client.h
+libgaim_client_la_LIBADD = $(DBUS_LIBS)
 
 gaim-client-bindings.c: dbus-analyze-functions.py $(dbus_exported)
 	cat $(dbus_build_exported) | $(PYTHON) $(srcdir)/dbus-analyze-functions.py --client > $@
