diff -urN gaim-0.44.org/src/Makefile.am gaim-0.44/src/Makefile.am
--- gaim-0.44.org/src/Makefile.am	Thu Sep 20 23:11:18 2001
+++ gaim-0.44/src/Makefile.am	Thu Sep 20 23:27:03 2001
@@ -63,27 +63,19 @@
 endif
 
 # UI stuff. We do this here so the PRPLs aren't forced to link with it
-LIBS += $(UI_LIBS)
-
 # Sound libs. Just for fun
-LIBS += $(SOUND_LIBS)
-
 # Static protocols
-LIBS += $(STATIC_LINK_LIBS)
-
 # Perl stuff. plugins shouldn't need this either (right?)
-CFLAGS += $(PERL_CFLAGS)
-LIBS += $(PERL_LIBS)
-
 # X screensaver extension
-LIBS += $(XSS_LIBS)
-
 # gettext stuff
-LIBS += $(INTLLIBS)
+# any other stuff
+LIBS = @LIBS@ $(UI_LIBS) $(SOUND_LIBS) $(STATIC_LINK_LIBS) $(PERL_LIBS) $(XSS_LIBS) $(INTLLIBS) $(DEBUG_LIBS)
+
+# Perl stuff. plugins shouldn't need this either (right?)
+CFLAGS += $(PERL_CFLAGS)
 
 # any other stuff
 CFLAGS += -DLOCALEDIR=\"$(datadir)/locale\" -DLIBDIR=\"$(libdir)/gaim/\" $(DEBUG_CFLAGS)
-LIBS += $(DEBUG_LIBS)
 
 EXTRA_DIST =	applet.h \
 		convo.h \
diff -urN gaim-0.44.org/src/protocols/zephyr/Makefile.am gaim-0.44/src/protocols/zephyr/Makefile.am
--- gaim-0.44.org/src/protocols/zephyr/Makefile.am	Thu Sep 20 23:11:18 2001
+++ gaim-0.44/src/protocols/zephyr/Makefile.am	Thu Sep 20 23:23:02 2001
@@ -2,7 +2,7 @@
 
 CFLAGS += -I\$(top_srcdir)/src -I\$(top_srcdir)/src/protocols -DCONFDIR=\"$(confdir)\" $(KRB4_CFLAGS) $(st) $(DEBUG_CFLAGS)
 
-LIBS += $(KRB4_LDFLAGS) $(KRB4_LIBS)
+LIBS = @LIBS@ $(KRB4_LDFLAGS) $(KRB4_LIBS)
 
 if STATIC_ZEPHYR
 
