--- pidgin-2.8.0/pidgin/plugins/gevolution/gevolution.c.old	2011-06-09 19:22:15.000000000 +0200
+++ pidgin-2.8.0/pidgin/plugins/gevolution/gevolution.c	2011-06-09 19:22:47.000000000 +0200
@@ -447,7 +447,7 @@ get_config_frame(PurplePlugin *plugin)
 	treeview = gtk_tree_view_new_with_model(GTK_TREE_MODEL(model));
 	gtk_tree_view_set_rules_hint(GTK_TREE_VIEW(treeview), TRUE);
 	gtk_box_pack_start(GTK_BOX(vbox),
-		pidgin_make_scrollable(treeview, GTK_POLICY_AUTO, GTK_POLICY_ALWAYS, GTK_SHADOW_IN, 300, 300),
+		pidgin_make_scrollable(treeview, GTK_POLICY_AUTOMATIC, GTK_POLICY_ALWAYS, GTK_SHADOW_IN, 300, 300),
 		TRUE, TRUE, 0);
 	gtk_widget_show(treeview);
 
